bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/widgets/ControlRenderer.java

```diff
@@ -18,15 +18,14 @@
 
 import java.util.function.Function;
 
-public abstract class ControlRenderer {
-
-	public abstract Point computeDefaultSize();
+public abstract class ControlRenderer <C extends Control> {
```

Comment:
Shouldn't it be:
```suggestion
public abstract class ControlRenderer <C extends CustomControl> {
```
?

Reply:
Let's continue it in #167

Reply:
No, because there are now custom controls that derive from Composite, not from CustomControl.